contracts/SimpleBank.sol

```diff
@@ -1,49 +1,48 @@
 /*
-    This exercise has been updated to use Solidity version 0.6.12
-    Breaking changes from 0.5 to 0.6 can be found here: 
-    https://solidity.readthedocs.io/en/v0.6.12/060-breaking-changes.html
+    This exercise has been updated to use Solidity version 0.8.5
+    See the latest Solidity updates at
+    https://solidity.readthedocs.io/en/latest/080-breaking-changes.html
 */
-
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
```

Comment:
I just checked out [the latest Solidity documentation about NatSpec comment](https://docs.soliditylang.org/en/v0.8.6/natspec-format.html) and they recommend triple slashes for comments `///`, thoughts?

Reply:
Sure, the recommendation is for NatSpec comments associated with keywords `contract`, `interface`, `function`, and `event` which can be processed to generate documentation. You'll notice the `SPDX` line in the link utilizes double slashes. 

I used the double slashes (personal choice) to indicate it is immediate instructions, or hints instead of code documentation that NatSpec should be.

Maybe there's a case for incorporating NatSpec, or maybe there isn't? I went  back and forth and and landed on having the option to ask students to create the NatSpec comments as an exercise.